covid19_sfbayarea/data/marin_scraper.py

```diff
@@ -0,0 +1,303 @@
+#!/usr/bin/env python3
+import csv
+import json
+import numpy as np
+from typing import List, Dict, Tuple
+from selenium import webdriver
+from bs4 import BeautifulSoup
+from urllib.parse import unquote_plus
+from datetime import datetime
+import re
+
+from .utils import get_data_model
+
+def get_county_data() -> Dict:
+	"""Main method for populating county data"""
+
+	url = 'https://coronavirus.marinhhs.org/surveillance'
+	model = get_data_model()
+
+	chart_ids = {"cases": "Eq6Es", "deaths": "Eq6Es", "tests": '2Hgir', "age": "VOeBm", "gender": "FEciW", "race_eth": "aBeEd"} 
+	# population totals and transmission data missing.
+	model['name'] = "Marin County"
+	model['update_time'] = datetime.today().isoformat()
+	# No actual update time on their website? They update most charts daily (so the isoformat is only partially correct.)
```

Comment:
Not sure if it makes sense to use the date in the updated column. It's certain that the updated column is not representative of the whole page (when I check now, the updated column for those two charts is 6/19, but the last data point on the "Daily Count of Positive Results and Total Tests for Marin County Residents by Test Date" chart is 6/15). 

One way around this would be to note the last updated date for each set of data in the output (aka each csv), but then there's a problem b/c the race/ethnicity data don't have any indication of updated dates whatsoever...

Reply:
> It's certain that the updated column is not representative of the whole page

Then my idea definitely sounds like a bad one! 😅 

> An option is to note the last updated date for each element in the scraper_output["series"]. Since the the race/ethnicity/age data that doesn't have any indication of updated dates whatsoever, I could just say the last updated date is N/A or missing. How's that?

My 2¢: I think what you’re already doing here is OK (except it would still be good to note in `meta_from_baypd`), and am not sure it’s worth finding a way to note the update time for each element of the dataset — especially since we don’t have that information in an especially consistent way.

Reply:
Thanks for your feedback! I'll make sure to note it accordingly.